pom.xml

```diff
@@ -482,30 +380,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.apache.felix</groupId>
-        <artifactId>maven-bundle-plugin</artifactId>
-        <version>3.5.0</version>
-        <extensions>true</extensions>
-        <configuration>
-          <instructions>
-            <_exportcontents>
-              io.cdap.plugin.cloud.vision.*
-            </_exportcontents>
-            <Embed-Dependency>*;inline=false;scope=compile</Embed-Dependency>
-            <Embed-Transitive>true</Embed-Transitive>
-            <Embed-Directory>lib</Embed-Directory>
-          </instructions>
-        </configuration>
-        <executions>
-          <execution>
-            <phase>package</phase>
-            <goals>
-              <goal>bundle</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
```

Comment:
why is this removed? This makes the build not include the plugins. The `target/.jar` size isn't 22Mb but 128Kb. And using the REST api to upload the .jar throws `<title>Error 400 (Bad Request)!!1</title>`. But even after adding back this part the `<title>Error 400 (Bad Request)!!1</title>` still remains.
I build the project with `mvn -Dmaven.test.skip=true package`, what can be the problem? Could you help please?

Reply:
I agree, this should not have been removed.

Reply:
👍 thx